starknet_devnet/util.py

```diff
@@ -243,3 +244,26 @@ def get_fee_estimation_info(tx_fee: int, gas_price: int):
 def warn(msg: str, file=sys.stderr):
     """Log a warning"""
     print(f"\033[93m{msg}\033[0m", file=file)
+
+
+class SuppressLogger:
+    """ctx mng to suppress logger"""
+
+    def __init__(self, logger):
+        self.logger = logger
+
+    def __enter__(self):
+        self.logger.disabled = True
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.logger.disabled = False
+
+
+client_spec = importlib.util.find_spec("services.external_api.client")
+assert client_spec
+
+# FeederGatewayClient is implemented in such a way that it logs and raises;
+# this suppresses the logging
+suppress_feeder_gateaway_client_logger = SuppressLogger(
```

Comment:
What's the difference between this and `contextlib.redirect_stderr(None)` that has been used so far?

Reply:
`contextlib.redirect_stderr(None)` somehow not works when used in ForkedOrigin methods.